query-languages/m/csv-document.md

```diff
@@ -51,3 +51,24 @@ Table.FromRecords({
     [OrderID = "1", Item = "Fishing rod"],
     [OrderID = "2", Item = "1 lb. worms"]
 ```
+
+## Example 2
+
+Process CSV text with multiple delimiter characters.
```

Comment:
```suggestion
Process CSV text with multiple delimiter characters. In this example, the 3rd parameter specifies the delimiter pattern `#|#` to use instead of the default.
```